src/main/java/edu/gatech/chai/VRDR/messaging/BaseMessage.java

```diff
@@ -514,4 +515,44 @@ public static <T extends Bundle> T parse(Class<T> tClass, IParser parser, InputS
         }
     }
 
+    public static List<? extends Bundle> parseBundleOfBundles(Class clazz, VRDRFhirContext ctx, String filePath, String bundleString) {
```

Comment:
This is a good start, but needs a few changes:

There doesn't need to be a Class clazz argument here, all message types should be parsed.  The return type should be List<BaseMessage>.  What you want to do is like the C# class:
1. Extract each bundle from the list of bundles string
2. Check the bundle type and parse based on type for any of the message type subclasses of BaseMessage
3. Assemble a list of BaseMessage, adding each message from step 2
4. Return the List<BaseMessage>